sonic-xcvrd/scripts/xcvrd

```diff
@@ -262,6 +263,19 @@ def del_port_sfp_dom_info_from_db(logical_port_name, int_tbl, dom_tbl):
             logger.log_error("This functionality is currently not implemented for this platform")
             sys.exit(NOT_IMPLEMENTED_ERROR)
 
+# recover missing sfp table entries if any
+def recover_missing_sfp_table_entries(sfp_util, int_tbl, stop=threading.Event()):
```

Comment:
Why the default parameter (`stop=threading.Event()`)? I don't think this is what you intended here.

Reply:
you are correct, revised.

Reply:
@jleveque please check the revised code, should be OK now.